### PR TITLE
UI: Fix keyboard options regression

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR fixes a UI regression on the keyboard options on the Deploy Instance wizard (restored labels form the PR https://github.com/apache/cloudstack/pull/6326 in which they were removed):

image

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?
Deploy Instance wizard -> Verify new text options for keyboard

### CODE REVIEW

**Comment by nvazquez** [comment]
@blueorangutan ui

**Comment by harikrishna-patnala** [comment]
verified LGTM

**Review by harikrishna-patnala — APPROVED** [review]

**Review by DaanHoogland — APPROVED** [review]
CLGTM

**Review by shwstppr — APPROVED** [review]

**Comment by shwstppr** [comment]
Verified on QA server

Screenshot from 2023-02-15 13-16-01

**Review by weizhouapache — APPROVED** [review]
verified OK

there is no translation for "French AZERTY keyboard"  when language changes

File: ui/public/locales/pt_BR.json [hunk 1]
```diff
@@ -874,6 +874,7 @@
 "label.isvolatile": "Vol\u00e1til",
 "label.items": "itens",
 "label.items.selected": "item(ns) selecionados",
+"label.japanese.keyboard": "Japanese keyboard",
```
Comment (stephankruggg) [inline ui/public/locales/pt_BR.json:877]:
```suggestion
"label.japanese.keyboard": "Teclado japon\u00eas",
```

File: ui/public/locales/pt_BR.json [hunk 2]
```diff
@@ -1471,6 +1472,7 @@
 "label.showing": "Exibindo",
 "label.shrinkok": "Encolhimento OK",
 "label.shutdown.provider": "Desabilitar provider",
+"label.simplified.chinese.keyboard": "Simplified Chinese keyboard",
```
Comment (stephankruggg) [inline ui/public/locales/pt_BR.json:1475]:
```suggestion
"label.simplified.chinese.keyboard": "Teclado chin\u00eas simplificado",
```

File: ui/public/locales/pt_BR.json [hunk 3]
```diff
@@ -1504,6 +1506,7 @@
 "label.ssh.port": "Porta SSH",
 "label.sshkeypair": "Novo par de chaves SSH",
 "label.sshkeypairs": "Par de chaves SSH",
+"label.standard.us.keyboard": "Standard (US) keyboard",
```
Comment (stephankruggg) [inline ui/public/locales/pt_BR.json:1509]:
```suggestion
"label.standard.us.keyboard": "Teclado padr\u00e3o (EUA)",
```

File: ui/public/locales/pt_BR.json [hunk 4]
```diff
@@ -1631,6 +1634,7 @@
 "label.type.id": "Tipo do ID",
 "label.ucs": "UCS",
 "label.udp": "UDP",
+"label.uk.keyboard": "UK keyboard",
```
Comment (stephankruggg) [inline ui/public/locales/pt_BR.json:1637]:
```suggestion
"label.uk.keyboard": "Teclado do Reino Unido",
```

File: ui/public/locales/nl_NL.json [hunk 5]
```diff
@@ -1016,6 +1017,7 @@
 "label.sharedmountpoint": "SharedMountPoint",
 "label.shrinkok": "Verklein OK",
 "label.shutdown.provider": "Schakel provider uit",
+"label.simplified.chinese.keyboard": "versimpeld Chinees toetsenbord",
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply ui/public/locales/nl_NL.json:1020]: It is not very dutch to capatilise common nounc and verbs, @stephankruggg . I would not apply this. It is considered shouting or self glorification.

File: ui/public/locales/nl_NL.json [hunk 6]
```diff
@@ -1042,6 +1044,7 @@
 "label.srx": "SRX",
 "label.ssh.key.pairs": "SSH sleutel paren",
 "label.sshkeypair": "nieuw SSH sleutelpaar",
+"label.standard.us.keyboard": "standaard (Amerikaans-US) toetsenbord",
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply ui/public/locales/nl_NL.json:1047]: nor this one

**Comment by weizhouapache** [comment]
This is not the first issue , maybe also not the last issue caused by #6326

File: ui/public/locales/nl_NL.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (stephankruggg) [reply ui/public/locales/nl_NL.json:1020]: Sorry, did not know that. In that case, I agree. The intention of my suggestion was to standardize the first letter as capitalized, as in other files. :smiley:

**Comment by nvazquez** [comment]
Thanks @stephankruggg I've applied your suggestions for portuguese
@blueorangutan ui

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**